packages/angular/src/app/pages/planning-task-list/planning-task-list.component.ts

```diff
@@ -53,15 +59,11 @@ export class PlanningTaskListComponent implements OnInit, OnDestroy {
       this.service.getTasks()
     ]).pipe(
       map(
-        ([filteredTasks, allTasks]) => { return { allTasks, filteredTasks } })
+        ([filteredTasks, allTasks]) => { return { allTasks, filteredTasks }  })
     );
-    this.dataSubscription = this.service.getTasks().subscribe((tasks) => {
-      this.tasks = tasks;
-    });
-  }
 
-  ngOnDestroy() {
-    this.dataSubscription.unsubscribe();
+    this.popupFullScreen = this.screen.isSmallScreen();
+    this.screenSubscription = this.screen.changed.subscribe(() => this.updatePopup());
```

Comment:
We subscribe to the screen service but don't unsubscribe when component destroyed

Reply:
Fixed